protocol/x/revshare/keeper/msg_set_marketmapper_rev_share_details_test.go

```diff
@@ -0,0 +1,55 @@
+package keeper_test
+
+import (
+	"testing"
+
+	testapp "github.com/dydxprotocol/v4-chain/protocol/testutil/app"
+	"github.com/dydxprotocol/v4-chain/protocol/x/revshare/keeper"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dydxprotocol/v4-chain/protocol/lib"
+	"github.com/dydxprotocol/v4-chain/protocol/testutil/constants"
+	"github.com/dydxprotocol/v4-chain/protocol/x/revshare/types"
+)
+
+func TestSetMarketMapperRevenueShareDetailsForMarket(t *testing.T) {
+	tests := map[string]struct {
+		// Msg
+		msg *types.MsgSetMarketMapperRevShareDetailsForMarket
+		// Expected error
+		expectedErr string
+	}{
+		"Success - Set revenue share details for market": {
+			msg: &types.MsgSetMarketMapperRevShareDetailsForMarket{
+				Authority: lib.GovModuleAddress.String(),
+				Params: types.MarketRevShareDetailsParams{
+					MarketId: constants.MarketId0,
+					MarketMapperRevShareDetails: &types.MarketMapperRevShareDetails{
+						ExpirationTs: 100,
+					},
+				},
+			},
+			expectedErr: "",
+		},
```

Comment:
Add failure cases for invalid authority and empty authority

Reply:
done!